content/docs/hooks-reference.md

```diff
@@ -508,3 +514,59 @@ For example a custom Hook that returned a `Date` value could avoid calling the `
 ```js
 useDebugValue(date, date => date.toDateString());
 ```
+
+### `useDeferredValue` {#usedeferredvalue}
+
+```js
+const [deferredValue] = useDeferredValue(value);
+```
+
+TODO: description
+
+### `useTransition` {#usetransition}
+
+```js
+const [isPending, startTransition] = useTransition();
+```
+
+TODO: description
+
+## Library Hooks {#library-hooks}
+
+The following Hooks are provided for library authors to integrate libraries deeply into the React model, and are not typically used in application code.
+
+### `useId` {#useid}
```

Comment:
This one is something you might need to use in application code a bit more often when you have disconnected labels/forms for example. Libraries might be able to hide it but not always. So probably worth promoting this higher and mention the label/form field case as an example explicitly so it's not confused for other use cases.